README.md

```diff
@@ -197,7 +197,8 @@ Haskell vectors in C:
 {-# LANGUAGE QuasiQuotes #-}
 {-# LANGUAGE TemplateHaskell #-}
 import qualified Language.C.Inline as C
-import qualified Data.Vector.Storable.Mutable as V
+import qualified Data.Vector.Storable as V
+import qualified Data.Vector.Storable.Mutable as M
```

Comment:
Can you change this to `as VM`? This is the convention we use in the rest of the code.